src/app/core/permissions/entity-permissions.service.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { TestBed } from "@angular/core/testing";
+
+import {
+  EntityPermissionsService,
+  OperationType,
+} from "./entity-permissions.service";
+import { SessionService } from "../session/session-service/session.service";
+import { User } from "../user/user";
+import { Entity } from "../entity/entity";
+import { EntityConfigService } from "../entity/entity-config.service";
+
+describe("EntityPermissionsService", () => {
+  let service: EntityPermissionsService;
+  const mockConfigService: jasmine.SpyObj<EntityConfigService> = jasmine.createSpyObj(
+    ["getEntityConfig"]
+  );
+  const mockSessionService: jasmine.SpyObj<SessionService> = jasmine.createSpyObj(
+    ["getCurrentUser"]
+  );
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        { provide: EntityConfigService, useValue: mockConfigService },
+        { provide: SessionService, useValue: mockSessionService },
+      ],
+    });
+    service = TestBed.inject(EntityPermissionsService);
+  });
+
+  it("should be created", () => {
+    expect(service).toBeTruthy();
+  });
+
+  it("should give permission if nothing is defined", () => {
+    mockConfigService.getEntityConfig.and.returnValue(null);
+    let permitted = service.userIsPermitted(Entity, OperationType.CREATE);
+    expect(permitted).toBeTrue();
+    mockConfigService.getEntityConfig.and.returnValue({
```

Comment:
that's actually two tests here ;-)

Reply:
I put both of these tests into one function because they both fall into the category `nothing is defined`. In the first case nothing is defined for the entity, in the second case, something is defined for the entity, but nothing is defined for this operation. 
Of course they can be split up but I did not want to make the test description to detailed.